test/identity-provider.test.js

```diff
@@ -33,11 +33,11 @@ describe('Identity Provider', function () {
     rmrf.sync(testKeysPath)
   })
 
-  describe('constructor', () => {
+  describe('constructor', (identity) => {
```

Comment:
`identity` doesn't belong here. Let's rather add an assert to L44 to check that identity is null or smth (ie. use it).

Reply:
Word. Thought that might not work - certainly clears up the issue, but didn't seem elegant to me, either.

Reply:
Looks good now 👍